src/main/java/org/jenkinsci/test/acceptance/plugins/logparser/LogParserOutputPage.java

```diff
@@ -0,0 +1,92 @@
+package org.jenkinsci.test.acceptance.plugins.logparser;
+
+import org.jenkinsci.test.acceptance.po.PageObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+/**
+ * The logparser plugin uses two frames.
+ *
+ * To check the content of the frames the selenium driver must be switched to the frames.
+ *
+ * @author Michael Engel.
+ */
+public class LogParserOutputPage extends PageObject {
+
+    /**
+     * Defines the different types of frames used by the parsed output of the logparser
+     */
+    public enum LogParserFrame {
+        SIDEBAR("sidebar"), CONTENT("content");
+
+        private String frameName;
+        LogParserFrame(String frameName) {
+            this.frameName = frameName;
+        }
+
+        public String getFrameName() {
+            return this.frameName;
+        }
+    }
+
+    /**
+     * Constructor.
+     *
+     * @param po The page of the logparser.
+     */
+    public LogParserOutputPage(PageObject po) {
+        super(po.injector, po.url(""));
+    }
+
+
+    // URL to return from the opened frame to the default window
+    private String restoreURL = null;
+
+    /**
+     * Opens the defined frame in the current window.
+     *
+     * @param frame Enum of the frame to open in the current window.
+     */
+    public void openFrameInWindow(LogParserFrame frame) {
```

Comment:
`openFrameInWindow`,  `restoreWindow`, `switchToFrame`, `switchToDefaultContent`, `switchToMainframe` are unused. Do you need that? Provided you do, I would like the page object to be redesigned so the fact the page is build from frames is encapsulated in that page object rather then exposed.